app/src/main/res/layout/activity_login.xml

```diff
@@ -1,15 +1,103 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="org.systers.mentorship.view.activities.LoginActivity">
 
-    <Button
-        android:id="@+id/btnLogin"
+    <android.support.v7.widget.AppCompatImageView
+        android:id="@+id/imageViewLogo"
         android:layout_width="wrap_content"
+        android:layout_height="160dp"
+        android:layout_marginBottom="8dp"
+        android:layout_marginEnd="8dp"
+        android:contentDescription="@string/logo"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.137"
+        app:srcCompat="@drawable/mentorship_system_logo" />
+
+    <android.support.design.widget.TextInputLayout
+        android:id="@+id/inputTextLoginUsername"
```

Comment:
don't use long names. We are following 100 column spaces pattern in .kt files.

Reply:
"100 column spaces pattern" I had to think a bit to understand what you meant by this. I can change that

Reply:
Ok, I'll do it like that

Reply:
`et` for edit text and `til` for input text layout? @m-murad